PennMobileShared/Dining/DiningAnalyticsViewModel.swift

```diff
@@ -108,43 +108,58 @@ public class DiningAnalyticsViewModel: ObservableObject {
 
     func populateAxesAndPredictions() {
         filterData()
+        
```

Comment:
Some comment documentation about the different methods we use would be great! We're starting to use a variety of methods so clarity becomes important here for future development.